### ISSUE TITLE
Cog status logger

### ISSUE THREAD

**Issue by paulmikulskis** [paulmikulskis/redditat#29]
Cog is the brain that powers the APIs and Services for us.

We want to be able to ask Cog some questions such as:
- who currently has scheduled Workflows in the system?
- how many scheduled Workflows are currently running?
- how many connections to Redis is Cog currently utilizing?
- what are the names of all the currently scheduled Workflows?

since this log will likely be large, we should emit it only once every 2 minutes or something.

simply use the `@yungsten/utils/logging` package, and some existing Cog functions such as `getScheduledWorkflows` to produce this log.

After this log is produced, you should notice that it is automatically picked up by Vector inside the Docker Container, and is then shipped to the Vector Aggregator at Yungsten Tech, where the log is stored (along with all the other logs) in [Grafana Loki](https://grafana.com/oss/loki/).  We can query Grafana Loki to see this log that we are producing above, and then create dashboards.

Grafana: https://grafana.yungstentech.com

Creating dashboards is not part of this issue, but it would be great if you want to try out Grafana and see how nifty it is to scan for data, make dashboards, etc, if you have not tried it before.
